.expeditor/end_to_end.pipeline.yml

```diff
@@ -858,6 +857,36 @@ steps:
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
 
+  - label: "[:linux: test_pkg_exporter_help]"
+    command:
+      - bash .expeditor/scripts/end_to_end/run_e2e_test.sh dev test_pkg_exporter_help
```

Comment:
Did you happen to omit the `test_pkg_exporter_help.ps1` file in this PR perhaps?

Reply:
No, I actually added [it](https://github.com/habitat-sh/habitat/blob/795cd2fc4a2d59e1d84b19c75e75488bbabafe75/test/end-to-end/test_pkg_exporters_help.ps1#L1) in a previous PR but then forgot to actually add it to the pipeline :sweat_smile: . It should be on master.

Reply:
Ahh, the reason I didn't find it on master is that it's named **exporters** : `./test/end-to-end/test_pkg_exporters_help.ps1`. In this PR it's referenced as `test_pkg_exporter_help`.

Reply:
Aha, nice catch, thanks! Addressed in [ad86cc](https://github.com/habitat-sh/habitat/pull/7865/commits/ad86ccc5c9828de1aa63c60e3ee65a8fe9e2d0a0)